index.html

```diff
@@ -18,12 +18,19 @@
             <span class="bar"></span>
           </div>
         </nav>
+        <div class="navbar-links">
+          <ul>
+            <li><a href="#">Poltfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+          </ul>
+        </div>
```

Comment:
- [ ] Great job on your portfolio! I noticed that the empty space on the right side of your page when switching from mobile to desktop view is still there. Some of the previous reviewers pointed this out too, it could be related to the **fixed widths** you're using in most of your elements. 

  To fix this, you might want to consider using a more responsive design approach, such as using **percentage-based widths** and **avoiding fixed values whenever possible**. I made a video to show you what I mean, keep up the good work! 💯

[scrnli_5_6_2023_7-02-07 PM.webm](https://user-images.githubusercontent.com/107952724/236651184-6229f3ca-4350-42bf-af3b-a8a4bb424bf3.webm)